.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict = true
```

Comment:
Needed?

Reply:
I added this to enforce yarn and restrict the use of npm